apps/web/src/components/commons/Input/index.stories.tsx

```
@@ -0,0 +1,39 @@
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+  args: {
+    id: "username",
+    label: "이름",
+    errorMessage: "",
+    type: "text",
+  },
+};
+
+export const Error: Story = {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)